elixir/apps/api/lib/api/client/views/relay.ex

```diff
@@ -5,31 +5,22 @@ defmodule API.Client.Views.Relay do
     Enum.flat_map(relays, &render(&1, expires_at, stun_or_turn))
   end
 
-  def render(%Relays.Relay{} = relay, expires_at, stun_or_turn) do
-    [
-      maybe_render(relay, expires_at, relay.ipv4, stun_or_turn),
-      maybe_render(relay, expires_at, relay.ipv6, stun_or_turn)
-    ]
-    |> List.flatten()
-  end
-
-  defp maybe_render(%Relays.Relay{}, _expires_at, nil, _stun_or_turn), do: []
-
   # STUN returns the reflective candidates to the peer and is used for hole-punching;
   # TURN is used to real actual traffic if hole-punching fails. It requires authentication.
   # WebRTC will automatically fail back to STUN if TURN fails,
   # so there is no need to send both of them along with each other.
 
-  defp maybe_render(%Relays.Relay{} = relay, _expires_at, address, :stun) do
+  def render(%Relays.Relay{} = relay, _expires_at, :stun) do
     [
       %{
         type: :stun,
-        addr: "#{format_address(address)}:#{relay.port}"
+        ipv4_addr: if(relay.ipv4, do: "#{format_address(relay.ipv4)}:#{relay.port}"),
+        ipv6_addr: if(relay.ipv6, do: "#{format_address(relay.ipv6)}:#{relay.port}")
```

Comment:
This is the change that you will see: `ipv4_addr` and `ipv6_addr` for each relay, either of those can be `null`.